rhoas/provider.go

```diff
@@ -13,39 +14,39 @@ import (
 )
 
 const (
-	DefaultAuthURL  = "https://sso.redhat.com/auth/realms/redhat-external"
-	DefaultMasAuthURL  = "https://identity.api.openshift.com/auth/realms/rhoas-client-prod"
-	DefaultApiUrl   = "https://api.openshift.com"
-	DefaultClientID = "cloud-services"
+	DefaultAuthURL    = "https://sso.redhat.com/auth/realms/redhat-external"
+	DefaultMasAuthURL = "https://identity.api.openshift.com/auth/realms/rhoas-client-prod"
+	DefaultApiUrl     = "https://api.openshift.com"
+	DefaultClientID   = "cloud-services"
 )
 
 // Provider -
 func Provider() *schema.Provider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
-			"offline_token": &schema.Schema{
+			"offline_token": {
 				Type:        schema.TypeString,
-				Optional:    true,
+				Optional:    false,
```

Comment:
I'm not sure this is valid, as it requires it to be in the schema If I Recall Correctly

Reply:
But that change is to make it non optional. I need to read more about plugins first and will fix it. Not sure what optional means right now

Reply:
Right - I think it's non-optional in that "you must put it in the .tf file always" so it can never hit the env var default.